### ISSUE TITLE
file.extension missing if index_content is set to false

### ISSUE THREAD

**Issue by kneubi** [dadoonet/fscrawler#317]
When i set index_content to false, the file.extension field is missing. the file.extension field is a very valuable filed and it might  be create if it would be stored regardless if content indexing is enabled or not.

**Comment by trorbyte** [comment]
I agree with Kneubi - having the Tika metadata populated without having to index content would be very useful. I'm trying to use fscrawler with geo files, the content is not really useful - but the metadata is.

**Comment by kneubi** [comment]
I have now modified the source code, to fix this issue. I changed to following parts:
- implemented a function getFileExtension in FsCrawlerUtil
- extended class FileAbstractModel with a string "extension"
- extended class FileAbstractorFile with model.extension = FsCrawlerUtil.getFileExtension(file);
- extended class FsCrawlerImpl with with a string "extension"
- added doc.getFile().setExtension(extension) to FsCrawlerImpl
- commented extension stuff in TikaDocParser.java

It works perfectly in my test environment, thus i will add the git diff output. I have no experience in Java coding, thus i am not sure if everything is implemented as it should and you will like the changes.

[file_extension_without_index_content.diff.zip](https://github.com/dadoonet/fscrawler/files/766567/file_extension_without_index_content.diff.zip)
